### PR TITLE
PWN-7614 - bridge send signin transaction

### PR DESCRIPTION
**Pull request by NikolaiEmelianov**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-7614

## Description of Work

bridge send signin transaction

### CODE REVIEW

File: app/src/main/java/org/p2p/wallet/bridge/send/BridgeSendInteractor.kt [hunk 1]
```diff
@@ -1,10 +1,59 @@
...
+            mint = mint,
+            amount = amount.toPlainString()
+        )
+        val userAccount = Account(tokenKeyProvider.keyPair)
+
+        val signedTransaction = relaySdkFacade.signTransaction(
+            transaction = sendTransaction.transaction.toBase58Instance(),
+            keyPair = userAccount.getEncodedKeyPair().toBase58Instance(),
+            // todo need it?
+            recentBlockhash = null
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/bridge/send/BridgeSendInteractor.kt:50]: if there is recentBlockhash already in sendTransaction then you don't need it 
if there is not - fetch it here and pass as a parameter

**Review by Davran331122 — APPROVED** [review]

File: app/src/main/java/org/p2p/wallet/bridge/send/BridgeSendInteractor.kt (hunk 1 above)
Comment (NikolaiEmelianov) [reply app/src/main/java/org/p2p/wallet/bridge/send/BridgeSendInteractor.kt:49]: recentBlockhash not need, deleted todo

**Review by kabramovp2p — APPROVED** [review]

File: app/src/main/java/org/p2p/wallet/bridge/send/BridgeSendInteractor.kt [hunk 2]
```diff
@@ -1,10 +1,105 @@
...
+        source: SolAddress,
+        recipient: EthAddress,
+        token: Token.Active,
+        amount: BigDecimal
+    ): String = withContext(dispatchers.io) {
+        val tokenMint = token.mintAddress
+        val userPublicKey = tokenKeyProvider.publicKey
+        val userWallet = SolAddress(userPublicKey)
+
+        val relayAccount = feeRelayerAccountInteractor.getUserRelayAccount()
```
Comment (1sakov) [inline app/src/main/java/org/p2p/wallet/bridge/send/BridgeSendInteractor.kt:50]: feeRelayerAccountInteractor.getRelayInfo().feePayerAddress

**Comment by 1sakov** [comment]
this will need to be changed when fee relayer will be updated

**Review by 1sakov — CHANGES_REQUESTED** [review]
this will need to be changed when fee relayer will be updated

**Review by 1sakov — APPROVED** [review]

**Pull request merged**